spec/requests/api/prosecution_cases_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
```

Comment:
if you want to remove the need to require this you can amend [.rspec](https://github.com/ministryofjustice/laa-court-data-adaptor/blob/master/.rspec) with this
```
--require rails_helper
```

Reply:
Good idea,
I made a subtask to move this in https://dsdmoj.atlassian.net/browse/CACP-165 (via a new PR so as to avoid this one getting large)

Reply:
Added via https://github.com/ministryofjustice/laa-court-data-adaptor/pull/63 
Thanks @mpw5